app/util/get-version.js

```diff
@@ -28,8 +28,9 @@ module.exports = function getVersion(pkg, options) {
   var nightly = options.nightly;
 
   if (nightly) {
-    appVersion = semver.inc(appVersion, 'minor') + '-' + nightly;
+    var timeStamp = (new Date()).getTime();
+    appVersion = semver.inc(appVersion, 'minor') + '-' + nightly + '-' + timeStamp;
```

Comment:
Can we separate this using a `.` to match [existing conventions](https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions), cf. [electron releases, too](https://github.com/electron/electron/releases?after=v6.1.6)?

Example: `v5.4.0-nightly.20191012` rather than `v5.4.0-nightly-20191012`.